src/rgw/rgw_dedup_store.cc

```diff
@@ -251,9 +257,12 @@ namespace rgw::dedup {
     stream << "num_parts = " << rec.s.num_parts << "\n";
     stream << "obj_size  = " << rec.s.obj_bytes_size/1024 <<" KiB"  << "\n";
     stream << "MD5       = " << std::hex << rec.s.md5_high << rec.s.md5_low << "\n";
-    stream << "SHA256    = ";
-    for (int i =0; i < 4; i++) {
-      stream << rec.s.sha256[i];
+    stream << "BLAKE3    = ";
```

Comment:
maybe better to name it "Hash" instead of "BLAKE3".
if this is a new RGW joining an already running dedup of old RGWs, the value might be sha256

also, in the future, we may replace blake3 with something else, and this will keep the output consistent

Reply:
I can change the name to hash, but we won't be able to mix different hash functions.
We also need to use the correct tag on the object

Reply:
i think it would be simpler for any future changes in the hash function.
e.g. the changes you had to make to the test code